app/models/external_utils/database_update_utils.rb

```diff
@@ -18,7 +18,7 @@ def self.convert_strings_to_json
               begin
                 simulation_run.result = Utils.parse_json_if_string(simulation_run.result)
                 simulation_run.cpu_info = Utils.parse_json_if_string(simulation_run.cpu_info)
-                simulation_run.tmp_result = Utils.parse_json_if_string(simulation_run.tmp_result)
+                simulation_run.tmp_results_list = Utils.parse_json_if_string(simulation_run.tmp_results_list)
```

Comment:
Nie musiałeś (właściwie to nie powinieneś) tu tego dopisywać, bo ten kod służył do konwersji starej bazy, a pole jest nowe. Przywróć linię simulation_run.tmp_result - nie powinna ona niczego psuć, a jeśli tmp_result się znajdzie w bazie to zostanie skonwertowane.